pkgs/tools/misc/yubikey-manager-qt/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, fetchurl
+, makeWrapper
+, pyotherside
+, pythonPackages
+, python3
+, qmake
+, qt5
+, yubikey-manager
+, yubikey-personalization
+}:
+
+with import <nixpkgs> {};
+
+let
+  qmlPath = qmlLib: "${qmlLib}/${qt5.qtbase.qtQmlPrefix}";
+
+  qml2ImportPath = lib.concatMapStringsSep ";" qmlPath [
+    qt5.qtbase.bin qt5.qtdeclarative.bin pyotherside qt5.qtquickcontrols qt5.qtquickcontrols2.bin qt5.qtgraphicaleffects
+  ];
+
+in stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

Comment:
Doesn't seem like pname is used anywhere else, just inline it in here?

Reply:
That seems to be a standard attribute, given what @Ma27 said about the `meta.name` attribute.

Reply:
I obviously didn't explain this well enough: you can either use `pname` (and `version`) which is concatenated to `${pname}-${version}` or simply use a `name` attribute. The first option has been implemented for convenience reasons, the latter is more commonly used.